pkg/cmd/util/env.go

```diff
@@ -91,6 +93,32 @@ func ParseEnvironmentArguments(s []string) (Environment, []string, []error) {
 	return env, duplicates, errs
 }
 
+func ParseEnvironmentArgumentsFiles(s []string) (Environment, []string, []error) {
```

Comment:
ok i see why you used a new name...  how about ParseEnvironmentArgumentsWithFileValues.  or update the existing helper to take a boolean arg that indicates if it should tolerate the \@file syntax or not.